authorization/role/repository/role_mapping_blackbox_test.go

```diff
@@ -132,3 +132,16 @@ func (s *roleMappingBlackBoxTest) TestOKToSave() {
 	require.Nil(s.T(), err, "Could not load role mapping")
 	require.Equal(s.T(), rm1.ResourceID, updatedRm.ResourceID)
 }
+
+func (s *roleMappingBlackBoxTest) TestFindForResource() {
+	g := s.NewTestGraph()
```

Comment:
Create some "noise" in the repo to make sure you found the expected mapping only.

Reply:
Done